environments/development/public.yml

```diff
@@ -107,3 +107,16 @@ commcare_cloud_remote_user: vagrant
 commcare_cloud_root_user: ubuntu
 commcare_cloud_strict_host_key_checking: no
 commcare_cloud_use_vault: no
+
+# A config should be specified for each host in `pgpool` host group
+pgpool_config:
```

Comment:
Another option is to put this in `postgresql.yml` and edit the schema in `schemas/postgresql.py` to accept this in that file and to auto-translate the host group aliases (like `db1`) to the host IP

Reply:
True, I considered that, but this turned out to be a bit easier for me to figure out.

Reply:
Doing that actually.